aws/resource_aws_msk_cluster.go

```diff
@@ -0,0 +1,307 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/kafka"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+	"time"
+)
+
+func resourceAwsMskCluster() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsMskClusterCreate,
+		Read:   resourceAwsMskClusterRead,
+		Delete: resourceAwsMskClusterDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Import functionality is missing acceptance testing. This can be accomplished by adding the below `TestStep` to every acceptance test:

```go
{
	ResourceName:      "aws_msk_cluster.test_cluster",
	ImportState:       true,
	ImportStateVerify: true,
},
```